pkgs/by-name/mi/mighty-mike/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  SDL2,
+  cmake,
+  makeWrapper,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mighty-mike";
+  version = "3.0.2-unstable-2024-11-10";
```

```suggestion
  version = "3.0.2-unstable-2024-04-01";
```
Should match the commit date, not the pull request date